lightning_client.go

```diff
@@ -4145,10 +4145,13 @@ func (s *lightningClient) SendCustomMessage(ctx context.Context,
 // are needed for verification.
 func (s *lightningClient) SignMessage(ctx context.Context,
 	data []byte) (string, error) {
-	rpcCtx := s.adminMac.WithMacaroonAuth(ctx)
 
+	rpcCtx := s.adminMac.WithMacaroonAuth(ctx)
 	rpcReq := &lnrpc.SignMessageRequest{Msg: data}
 	rpcRes, err := s.client.SignMessage(rpcCtx, rpcReq)
+	if err != nil {
+		return "", err
+	}
 
 	return rpcRes.Signature, err
```

Comment:
Should now just return `rpcRes.Signature, nil` here.

Reply:
Sharp eyed reviewer! Fixed.